### DISCUSSION TITLE
Diango 4.2.16 does not work since update to mysqlclient-2.2.5-cp312-cp312-win_amd64.whl due to 'TLS/SSL error'

### DISCUSSION THREAD

**Discussion by stephanema1** [PyMySQL/mysqlclient#735] [Issue report]
### Describe the bug

I have:
- Win10
- python 3.12.7
- django 4.2.16 (djangoproject.org)
- MariaDB 10.11.9 (mariadb-10.11.9-winx64.msi)

all worked fine, but yesterday I updated my packages
`mysqlclient-2.2.4-cp312-cp312-win_amd64.whl` to `mysqlclient-2.2.5-cp312-cp312-win_amd64.whl`
and from now on, when I call a webpage of one of my django websites I get the following error:
 
    File "C:\tooling\programs\python312\Lib\site-packages\MySQLdb\connections.py", line 195, in __init__
        super().__init__(*args, **kwargs2)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Exception Type: OperationalError at /mywebsite/ Exception Value: (2026, 'TLS/SSL error: SSL is required, but the server does not support it')

So I downgraded to `mysqlclient-2.2.4-cp312-cp312-win_amd64.whl` and all is working fine again.

I didn't find any breaking "hints" in the mysqlclient changelogs or at
https://github.com/mariadb-corporation/mariadb-connector-c (they have no changelogs at all).

I suppose some settings are somewhere needed to prevent some new behaviour ?

### How did you install libmysqlclient libraries?

pip

### What version of mysqlclient do you use?

2.2.5

### Minimum but complete code to reproduce

```python
# Write Python code here.
import MySQLdb

conn = MySQLdb.connect(host='127.0.0.1', port=3306, user='root')
...
```

### Schema and initial data required to reproduce.

```sql
-- Write SQL here.
-- e.g. CREATE TABLE ...
```

**Comment by methane** [comment]
duplicate of #729.
See [this comment](https://github.com/PyMySQL/mysqlclient/discussions/729#discussioncomment-11035085).
